services/frontendhost/frontend_test.go

```diff
@@ -663,6 +663,72 @@ func (s *FrontendHostSuite) TestFrontendHostCreateConsumerGroup() {
 	}
 }
 
+// TestFrontendHostCreateConsumerGroupStartFrom tests whether CreateConsumerGroup is able to
+// detect and appropriately convert various startFrom units.
+func (s *FrontendHostSuite) TestFrontendHostCreateConsumerGroupStartFrom() {
+
+	testPath := uuid.New()
+	testCG := s.generateKey("/bar/CGName")
+	frontendHost, ctx := s.utilGetContextAndFrontend()
+
+	// set start-from to, say, four weeks ago
+	startFromTime := time.Now().Add(-4 * 7 * 24 * time.Hour)
```

Comment:
i am not sure what you mean by the boundary .. but i think we need to test whether this works okay with dates within "reasonable" boundaries. i'll add some more.